torchrl/objectives/a2c.py

```diff
@@ -67,24 +69,57 @@ class A2CLoss(LossModule):
 
     """
 
+    @dataclass
+    class _AcceptedKeys:
+        """Maintains default values for all configurable tensordict keys.
+
+        This class defines which tensordict keys can be set using '.set_keys(key_name=key_value)' and their
+        default values.
+
+        Attributes:
+        ------------
+        advantage : NestedKey
+            The input tensordict key where the advantage is expected.
```

Comment:
Should follow the Google docstring formatting

```suggestion
        Attributes:
            advantage (NestedKey): The input tensordict key where the advantage is expected.
```